npm_and_yarn/lib/dependabot/npm_and_yarn/file_parser/lockfile_parser.rb

```diff
@@ -190,7 +190,11 @@ def semver_version_for(version_string)
         end
 
         def alias_package?(requirement)
-          Experiments.enabled?(:yarn_berry) ? requirement.match?(/@npm:(.+@(?!npm))/) : requirement.include?("@npm:")
+          if Experiments.enabled?(:yarn_berry)
+            requirement.match?(/@npm:(.+@(?!npm))/)
+          else  
```

Comment:
Sorry, I had a space in my suggestion and it's making Rubocop :rage4: 
```suggestion
          else
```